doc/qbk/1.1.quicklook.qbk

```diff
@@ -13,6 +13,8 @@ This section is intended to give the reader a brief overview of the features
 and interface style of the library.
 
 Say you have the following URL that you want to parse:
+
+[teletype]
```

Comment:
why do we need this 'teletype' ?

Reply:
Yes. Without it, quickbook tries to interpret that as C++ and, besides the ugly attempt to syntax highlight that as C++, we get

```
libs/url/doc/qbk/1.1.quicklook.qbk:18: warning: in column:22, unexpected character: @
libs/url/doc/qbk/1.1.quicklook.qbk:49: warning: in column:26, unexpected character: @
```

Reply:
The complete output would be:

```
b2 libs/url/doc -a | egrep -i -B 1 -A 2 '(warning|error)\s*:'
    inlined from ‘void quickbook::quickbook_grammar::impl::init_phrase_elements()’ at /home/alandefreitas/Documents/Code/C++/boost/tools/quickbook/src/phrase_element_grammar.cpp:41:65:
/home/alandefreitas/Documents/Code/C++/boost/tools/quickbook/src/actions.hpp:184:12: warning: ‘<unnamed>.quickbook::cond_phrase_push::saved_conditional’ may be used uninitialized [-Wmaybe-uninitialized]
  184 |     struct cond_phrase_push : scoped_action_base
      |            ^~~~~~~~~~~~~~~~
--
Generating output file: bin.v2/libs/url/doc/url_doc.xml
libs/url/doc/qbk/1.1.quicklook.qbk:18: warning: in column:22, unexpected character: @
libs/url/doc/qbk/1.1.quicklook.qbk:49: warning: in column:26, unexpected character: @
bin.v2/libs/url/doc/reference.qbk:357: warning: line breaks generate invalid boostbook (will only note first occurrence).
xslt-xsltproc bin.v2/libs/url/doc/url_doc.docbook
xslt-xsltproc-dir libs/url/doc/html/url_HTML.manifest
...updated 189 targets...
```